### PR TITLE
Update tiflash-configuration.md

### PR DESCRIPTION
**Pull request by flowbehappy**
Added documents for the following settings.

* store limit
* mark_cache_size
* minmax_index_cache_size
* dt_enable_logical_split
* max_memory_usage_for_all_queries

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3491
- Other reference link(s):

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ This document introduces the configuration parameters related to the deployment
...
 
     > **Notes:**
     >
     > The value of this parameter should be less than that of `region-schedule-limit`. Otherwise, the normal Region scheduling among TiKV nodes is affected.
 
-- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which each store is scheduled.
+- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the region balance rate at which each store (i.e. TiKV or TiFlash) is scheduled. Note that modifying this setting only works for later added stores. If you would like to modify current sotres', use the following command.
+    - Use command `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` to modify a specific store's region balance rate immediately. Note that this setting inherits `store-balance-rate` if you didn't specified. You can use `pd-ctl -u <pd_ip:pd_port> store limit` to check the current setting. And `store_id` can be found by `pd-ctl -u <pd_ip:pd_port> store`.
```
Comment (toutdesuite) [inline tiflash/tiflash-configuration.md:23]:
```suggestion
    - Execute the `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` command to set the scheduling rate of a specified Region. (To get `store_id`, you can execute the `pd-ctl -u <pd_ip:pd_port> store` command. 
    - If you do not set the scheduling rate of the specified Region, this Region inherits the value of `store-balance-rate`. 
    - You can execute the `pd-ctl -u <pd_ip:pd_port> store limit` command to view the value of `store-balance-rate`.
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -11,15 +11,16 @@ This document introduces the configuration parameters related to the deployment
...
+You can adjust the PD scheduling parameters using [pd-ctl](/pd-control.md). Note that you can use `tiup ctl pd` to replace `pd-ctl -u <pd_ip:pd_port>` if using tiup to deploy and manage your cluster.
 
 - [`replica-schedule-limit`](/pd-configuration-file.md#replica-schedule-limit): determines the rate at which the replica-related operator is generated. The parameter affects operations such as making nodes offline and add replicas.
 
     > **Notes:**
     >
     > The value of this parameter should be less than that of `region-schedule-limit`. Otherwise, the normal Region scheduling among TiKV nodes is affected.
 
-- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which each store is scheduled.
+- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the region balance rate at which each store (i.e. TiKV or TiFlash) is scheduled. Note that modifying this setting only works for later added stores. If you would like to modify current sotres', use the following command.
```
Comment (toutdesuite) [inline tiflash/tiflash-configuration.md:22]:
```suggestion
- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which Regions of each TiKV/TiFlash store are scheduled. Note that this parameter takes effect only when the stores have newly joined the cluster.
```

**Comment by flowbehappy** [comment]
@toutdesuite PTAL

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -11,15 +11,18 @@ This document introduces the configuration parameters related to the deployment
...
 
 - [`replica-schedule-limit`](/pd-configuration-file.md#replica-schedule-limit): determines the rate at which the replica-related operator is generated. The parameter affects operations such as making nodes offline and add replicas.
 
     > **Notes:**
     >
     > The value of this parameter should be less than that of `region-schedule-limit`. Otherwise, the normal Region scheduling among TiKV nodes is affected.
 
-- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which each store is scheduled.
+- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which Regions of each TiKV/TiFlash store are scheduled. Note that this parameter takes effect only when the stores have newly joined the cluster. If you would like to change that setting for present stores, use the command below.
+    - Execute the `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` command to set the scheduling rate of a specified Region. (To get `store_id`, you can execute the `pd-ctl -u <pd_ip:pd_port> store` command.
```
Comment (toutdesuite) [inline tiflash/tiflash-configuration.md:23]:
```suggestion
    - Execute the `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` command to set the scheduling rate for Regions of a specified store. (To get `store_id`, you can execute the `pd-ctl -u <pd_ip:pd_port> store` command. 
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -11,15 +11,18 @@ This document introduces the configuration parameters related to the deployment
...
 
     > **Notes:**
     >
     > The value of this parameter should be less than that of `region-schedule-limit`. Otherwise, the normal Region scheduling among TiKV nodes is affected.
 
-- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which each store is scheduled.
+- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which Regions of each TiKV/TiFlash store are scheduled. Note that this parameter takes effect only when the stores have newly joined the cluster. If you would like to change that setting for present stores, use the command below.
+    - Execute the `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` command to set the scheduling rate of a specified Region. (To get `store_id`, you can execute the `pd-ctl -u <pd_ip:pd_port> store` command. 
+    - If you do not set the scheduling rate of the specified Region, this Region inherits the value of `store-balance-rate`.
```
Comment (toutdesuite) [inline tiflash/tiflash-configuration.md:24]:
```suggestion
    - If you do not set the scheduling rate for Regions of a specified store, this store inherits the setting of `store-balance-rate`. 
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -11,15 +11,18 @@ This document introduces the configuration parameters related to the deployment
...
 
     > **Notes:**
     >
     > The value of this parameter should be less than that of `region-schedule-limit`. Otherwise, the normal Region scheduling among TiKV nodes is affected.
 
-- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which each store is scheduled.
+- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which Regions of each TiKV/TiFlash store are scheduled. Note that this parameter takes effect only when the stores have newly joined the cluster. If you would like to change that setting for present stores, use the command below.
+    - Execute the `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` command to set the scheduling rate of a specified Region. (To get `store_id`, you can execute the `pd-ctl -u <pd_ip:pd_port> store` command. 
+    - If you do not set the scheduling rate of the specified Region, this Region inherits the value of `store-balance-rate`. 
+    - You can execute the `pd-ctl -u <pd_ip:pd_port> store limit` command to view the value of `store-balance-rate`.
```
Comment (toutdesuite) [inline tiflash/tiflash-configuration.md:25]: Please double check whether "查看当前设置值" and "view the value of `store-balance-rate`" mean the same thing

File: tiflash/tiflash-configuration.md [hunk 6]
```diff
@@ -11,15 +11,18 @@ This document introduces the configuration parameters related to the deployment
 
 ## PD scheduling parameters
 
-You can adjust the PD scheduling parameters using [pd-ctl](/pd-control.md) (the binary file in `resources/bin` in the tidb-ansible directory):
+You can adjust the PD scheduling parameters using [pd-ctl](/pd-control.md). Note that you can use `tiup ctl pd` to replace `pd-ctl -u <pd_ip:pd_port>` when using tiup to deploy and manage your cluster.
```
Comment (toutdesuite) [inline tiflash/tiflash-configuration.md:14]:
```suggestion
You can adjust the PD scheduling parameters using [pd-ctl](/pd-control.md). Note that you can use `tiup ctl pd` to replace `pd-ctl -u <pd_ip:pd_port>` when using tiup to deploy the cluster.
```

File: tiflash/tiflash-configuration.md (hunk 3 above)
Comment (flowbehappy) [reply tiflash/tiflash-configuration.md:23]: I think `for Regions` is a little redundant.

Comment (toutdesuite) [reply tiflash/tiflash-configuration.md:23]: updated~

File: tiflash/tiflash-configuration.md (hunk 5 above)
Comment (flowbehappy) [reply tiflash/tiflash-configuration.md:25]: Checked and confirmed. And maybe "view the current setting value of `store-balance-rate`" is better?

File: tiflash/tiflash-configuration.md [hunk 7]
```diff
@@ -11,15 +11,18 @@ This document introduces the configuration parameters related to the deployment
...
 
     > **Notes:**
     >
     > The value of this parameter should be less than that of `region-schedule-limit`. Otherwise, the normal Region scheduling among TiKV nodes is affected.
 
-- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which each store is scheduled.
+- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which Regions of each TiKV/TiFlash store are scheduled. Note that this parameter takes effect only when the stores have newly joined the cluster. If you want to change the setting for existing stores, use the following command.
+    - Execute the `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` command to set the scheduling rate of a specified store. (To get `store_id`, you can execute the `pd-ctl -u <pd_ip:pd_port> store` command. 
+    - If you do not set the scheduling rate of the specified Region, this Region inherits the value of `store-balance-rate`. 
+    - You can execute the `pd-ctl -u <pd_ip:pd_port> store limit` command to view the value of `store-balance-rate`.
```
Comment (toutdesuite) [inline tiflash/tiflash-configuration.md:25]:
```suggestion
    - You can execute the `pd-ctl -u <pd_ip:pd_port> store limit` command to view the current setting value of `store-balance-rate`.
```

File: tiflash/tiflash-configuration.md (hunk 5 above)
Comment (toutdesuite) [reply tiflash/tiflash-configuration.md:25]: updated

**Pull request merged**